R/bulk_dl.R

```diff
@@ -46,7 +46,7 @@ bulk_dl <- function(data_set, output_dir = NULL, file_name = NULL, size_warning
     file_name <- paste0(data_set, ".csv")
   }
 
-  if (size_warning == T) {
+  if (size_warning) {
```

Comment:
here you may also want to check if the code is run interactively, so you don't wait for an answer if the code is executed in a non-interactive script
```suggestion
  if (size_warning && interactive()) {
```